v1/build_config.go

```diff
@@ -179,5 +180,6 @@ type bcCreateWrapper struct {
 	Version struct {
 		Metadata map[string]interface{} `json:"metadata,omitempty"`
 		Builds   []BuildConfigBuild     `json:"builds"`
+		Vars     map[string]string      `json:"vars"`
```

Comment:
Done! I think this is good because it forces Atlas to handle old and new clients, instead of always expecting a value there.